kube_apiserver_metrics/tests/test_kube_apiserver_metrics_1_25.py

```diff
@@ -0,0 +1,115 @@
+# (C) Datadog, Inc. 2023-present
+# All rights reserved
+# Licensed under a 3-clause BSD style license (see LICENSE)
+
+# stdlib
+import os
+
+import mock
+import pytest
+
+from datadog_checks.kube_apiserver_metrics import KubeAPIServerMetricsCheck
+
+customtag = "custom:tag"
+
+instance = {
+    'prometheus_url': 'https://localhost:443/metrics',
+    'bearer_token_auth': 'false',
+    'tags': [customtag],
+}
+
+
+@pytest.fixture()
+def mock_get():
+    f_name = os.path.join(os.path.dirname(__file__), 'fixtures', 'metrics_1.24.0.txt')
```

Comment:
the file name say Kube version 1.25, but the metrics file is 1.24. Is it mandatory to use 1.25?